app/views/notes/_title_suggestion.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="alert alert-warning">
-	<a href="/profile/<%= user %>"><%= user %></a> is suggesting an alternative title, "<%= title %>".
+        <a href="/profile/<%= user %>"><%= user %></a> <%= translation('notes._title_suggestion.suggesting_title') %> "<%= title %>".
```

Comment:
Nice @Angelvicks! Do you mind changing `translation` to `t` and indenting line 2 to match the other lines? Thanks!

Reply:
The indentation is well placed when I am editing But later when I commit it changes. So I don't know how to fix it.